xpk.py

```diff
@@ -71,6 +71,9 @@
 _LOCAL_QUEUE_NAME='multislice-queue'
 _CLUSTER_RESOURCES_CONFIGMAP = 'resources-configmap'
 _CLUSTER_METADATA_CONFIGMAP = 'metadata-configmap'
+_XPK_SERVICE_ACCOUNT = 'xpk-sa'
+# Set to True to attach a service account to cluster & node pools
+_USE_SERVICE_ACCOUNT = False
```

Comment:
_SERVICE_ACCOUNT_FEATURE_FLAG = Flase

seems like a nicer name? Since this flag means we attempt to attach a SA but we don't always do it (like for previously created clusters).

Reply:
I think this flag is great though, we can ultimately set it to = (xpk_version >= .4.0 ) or something.

Reply:
Yes, this is a great suggestion. Updated!